src/rprblender/nodes/node_parser.py

```diff
@@ -246,16 +246,11 @@ def export(self):
 
     def export_muted(self):
         # export as a muted node
-        # pass through first instance of same socket type
+        # pass through first linked socket of same output socket type
         matching_incoming_socket = next((input for input in self.node.inputs
-                    if isinstance(input, type(self.socket_out))), None)
+            if isinstance(input, type(self.socket_out)) and input.is_linked), None)
         if matching_incoming_socket:
-            if matching_incoming_socket.is_linked:
-                # pass through first incoming link if there is one
-                return self.get_input_link(matching_incoming_socket.name)
-            else:
-                # return black
-                return (0.0, 0.0, 0.0, 0.0)
+            return self.get_input_link(matching_incoming_socket.name)
```

Comment:
So my testing showed that in cycles if there was a matching socket but it wasn't linked it would return black. 

In the above code I think it would us the input value?  Correct me if I'm wrong.

Reply:
Hmm, my tests in cycles showed me that if nothing is connected to matching socket in muted node, than it works like nothing is connected (like muted node doesn't exists) and it uses default value.

Reply:
It's a bit of a weird use case anyway.  Ok.